docker/flink/Dockerfile

```diff
@@ -5,8 +5,11 @@ RUN wget -nv -P $FLINK_HOME/lib/ https://repo1.maven.org/maven2/org/apache/flink
 
 # install python3 and pip3
 RUN apt-get update -y && \
-apt-get install -y python3 python3-pip python3-dev && rm -rf /var/lib/apt/lists/*
+    apt-get install -y python3 python3-pip python3-dev openjdk-11-jdk-headless && \
+    rm -rf /var/lib/apt/lists/*
 RUN ln -s /usr/bin/python3 /usr/bin/python
 
 # install PyFlink
+ARG BUILDARCH
+ENV JAVA_HOME=/usr/lib/jvm/java-11-openjdk-${BUILDARCH:-amd64}
```

Comment:
Can this be computed with a reasonable shell command?

Reply:
Yes, but what is the problem with this? Buildkit automatically passed the `BUILDARCH` build argument.

Reply:
Ah, sweet